### ISSUE TITLE
scheduler.py: float division by zero

### ISSUE THREAD

**Issue by Joshuaalbert** [lofar-astron/factor#177]
Here is a traceback that halts factor

```
INFO - factor:directions - Adjusting facets to avoid sources...
/net/para10/data1/shimwell/pipeline/Pre-Facet-Cal/factor/factor/directions.py:808: RuntimeWarning: invalid value encountered in sqrt
  min_sizes = [fwhm*min(10.0, max(2.0, np.sqrt(flux_jy/0.01))) for flux_jy in fluxes_jy]
INFO - factor - Self calibrating 1 direction(s) in Group 1
Traceback (most recent call last):
  File "/net/para10/data1/shimwell/pipeline/Pre-Facet-Cal/factor/bin/runfactor", line 70, in <module>
    reset_operations)
  File "/net/para10/data1/shimwell/pipeline/Pre-Facet-Cal/factor/factor/process.py", line 176, in run
    scheduler.run(ops)
  File "/net/para10/data1/shimwell/pipeline/Pre-Facet-Cal/factor/factor/lib/scheduler.py", line 283, in run
    self.allocate_resources()
  File "/net/para10/data1/shimwell/pipeline/Pre-Facet-Cal/factor/factor/lib/scheduler.py", line 125, in allocate_resources
    for i in range(int(np.ceil(len(operation_list)/float(nops_simul)))):
ZeroDivisionError: float division by zero
```

**Comment by darafferty** [comment]
Have you specified a `clusterdesc_file` in your parset and, if so, what is it set to? It seems that for some reason it determines the number of nodes to be zero (since the denominator above, `nops_simul`, is determined as `nops_simul = num_nodes * ndir_per_node` and `num_nodes` depends on the `clusterdesc_file`)

**Comment by Joshuaalbert** [comment]
This is what is says. It's called local.clusterdesc in my pipeline dir. If I leave 
`clusterdesc_file = PBS` it will work with defaults all nodes on the PBS system?
```
ClusterName = Local
NNodes = 1
Node0.NodeName = localhost
Node0.NodeFileSys = localhost 
Node0.NodeMountPoints = /
```

**Comment by darafferty** [comment]
> This is what is says. It's called local.clusterdesc in my pipeline dir

OK -- for some reason, this file does not parse correctly with the `lofarpipe.support.clusterdesc` module and returns an empty list for the nodes. If you want to use PBS-reserved nodes, then yes, `clusterdesc_file = PBS` should work. Otherwise, just remove the `clusterdesc_file` entry from your parset to use a single (local) machine.

**Comment by Joshuaalbert** [comment]
Using PBS seems to work! This also solved issue #173
